qa/L0_sequence_batcher/sequence_batcher_test.py

```diff
@@ -2052,6 +2061,12 @@ def test_queue_delay_no_min_util(self):
         # execution to be a batch of 'null, seq 2'. The executions should not be
         # waited.
 
+        # Adjust the values of 'sequence_thresholds' due to Jetson intermittence
+        if TEST_JETSON:
+            sequence_thresholds = (6000, None)
```

Comment:
2000 -> 6000 is a big jump
I don't see why we need to set the lt_ms part of the sequence thresholds to a different value here when we already multiply it by the _jetson_slowdown_factor if the platform is jetson.

Reply:
The reason why I changed this is because the response time becomes really long when running this on jetson. (i.e. 13000 ms while originally it should be less than 4000 ms) I didn't make lots of changes on '_jetson_slowdown_factor' is because the variable will affect other tests, but I agree that we should modify only this variable instead of the sequence thresholds since the latter one is already multiplied by the factor. Will modify.

Reply:
_jetson_slowdown_factor is only used inside sequence_util.py